ALSpringBackend/src/main/java/com/ardaslegends/data/domain/SpecialBuilding.java

```diff
@@ -0,0 +1,14 @@
+package com.ardaslegends.data.domain;
+
+public enum SpecialBuilding {
```

Comment:
Check comment in ProdBuilding enum, they are listed as prod sites so I would ask glupers/brack

Reply:
Refer to the other ProdType comment.